### PR TITLE
Module/liftover/1.1

### PR DESCRIPTION
**Pull request by Kdreval**
# Pull Request Checklists

**Important:** When opening a pull request, keep only the applicable checklist and delete all other sections.

## Checklist for Updated Module

This update includes automatic filtering for hg38 samples and also option to include name of parent directory for output. The latter allows tracking of not only variant caller tool, but also version the module version. The most recent master was merged  into this PR.

### CODE REVIEW

File: demo/Snakefile [hunk 1]
```diff
@@ -34,7 +34,7 @@ configfile: "../modules/sequenza/1.4/config/default.yaml"
 configfile: "../modules/strelka/1.1/config/default.yaml"
 configfile: "../modules/utils/2.0/config/default.yaml"
```
Comment (rdmorin) [inline demo/Snakefile]: Can you remove liftOver from the demo and sync this PR up with Master?

**Review by rdmorin — CHANGES_REQUESTED** [review]

File: demo/Snakefile (hunk 1 above)
Comment (Kdreval) [reply demo/Snakefile]: Just did it

**Comment by rdmorin** [comment]
@Kdreval can you look into the conflicts on this?

**Comment by Kdreval** [comment]
I have merged most recent master and resolved merging conflicts. Can confirm dry-run of demo project runs without errors (and my next step is working on stemming demo project)

**Review by rdmorin — APPROVED** [review]

**Pull request merged**